articles/container-service/container-service-connect.md

```diff
@@ -71,53 +71,64 @@ For further instructions you can see the [Kubernetes quick start](http://kuberne
+
+This document walks you through creating an SSH tunnel from Linux, OS X, and Windows clients.
 
 > [!NOTE]
-> You can create an SSH session with a cluster management system. However, we don't recommend this. Working directly on a management system exposes the risk for inadvertent configuration changes.   
+> These instructions focus on tunnelling TCP traffic over SSH. You can also start an interactive SSH session with one of the internal cluster management systems, but we don't recommend this. Working directly on an internal system risks inadvertent configuration changes.   
 > 
 > 
 
 ## Create an SSH tunnel on Linux or OS X
-The first thing that you do when you create an SSH tunnel on Linux or OS X is to locate the public DNS name of load-balanced masters. To do this, expand the resource group so that each resource is being displayed. Locate and select the public IP address of the master. This will open up a blade that contains information about the public IP address, which includes the DNS name. Save this name for later use. <br />
 
-![Public DNS name](media/pubdns.png)
+1. Get the public DNS name of the load balancer for the master group.
 
-Now open a shell and run the following command where:
+  * To do this in the portal, expand the resource group so that each resource is being displayed. Locate and select the public IP address of the master. This will open up a blade that contains information about the public IP address, which includes the DNS name. Save this name for later use. <br />
 
-**PORT** is the port of the endpoint that you want to expose. For Swarm, this is 2375. For DC/OS, use port 80.  
-**USERNAME** is the user name that was provided when you deployed the cluster.  
-**DNSPREFIX** is the DNS prefix that you provided when you deployed the cluster.  
-**REGION** is the region in which your resource group is located.  
-**PATH_TO_PRIVATE_KEY** [OPTIONAL] is the path to the private key that corresponds to the public key you provided when you created the Container Service cluster. Use this option with the -i flag.
+  ![Public DNS name](media/pubdns.png)
+
+  * Alternatively, use the `az acs show ...` command and look for the "Master Profile:fqdn" property.
+
+2. Run the following command to create an SSH tunnel with one of the cluster
+   masters.  
 
 ```bash
-ssh -L PORT:localhost:PORT -f -N [USERNAME]@[DNSPREFIX]mgmt.[REGION].cloudapp.azure.com -p 2200
+ssh -fNL LOCAL_PORT:localhost:REMOTE_PORT -p 2200 [USERNAME]@[DNSPREFIX]mgmt.[REGION].cloudapp.azure.com
 ```
-> The SSH connection port is 2200--not the standard port 22.
-> 
-> 
+
+**LOCAL_PORT** is the local TCP port to be forwarded over the tunnel.  
```

Comment:
If a port other than 80 is used for DC/OS is used then some links in the UI will break. If we are going to avoid instructing people to use port 80 then we should note this (it's very few links these days but still an issue).